client/dashboard/components/callout/index.tsx

```diff
@@ -12,12 +12,12 @@ import './styles.scss';
 function UnforwardedCallout(
 	{
 		title,
+		titleAs: TitleComponent = Text,
```

Comment:
How about simply calling it `TitleComponent`?

Reply:
Or remove the default value of the `headingLevel`. If the `headingLevel` is not provided, we could use the `Text` component. Is it a good idea 🤔

Reply:
I think I've seen the convention of using the `as` prop name for this sort of thing. `<Text>` and `<View>` from core components use it. Riakit too.

I think the reason they don't just call it `Component` is that "h1" or "div" aren't technically components iirc.

So I was technically wrong to rename `titleAs` to `TitleComponent` I think. It just has to be capitalised to satisfy the JSX compiler. I should probably do `titleAs: Title = Text`, so that when it gets used it looks like `<Title>{ title }</Title>`